bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZZTo4l0To3jets/ZZNJets_Zll_VBF_LO_run_card.dat

```diff
@@ -0,0 +1,269 @@
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  ZZNJets_Zll_VBF_LO     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+ .true.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ lhapdf = pdlabel ! PDF set
+ 316200 = lhaid
```

Comment:
Is there a reason to use this pdf instead of the standard/recommended pdf?

Reply:
@sihyunjeon  it was a suggestion from PdmV group in the beginning of 2020. This was the most recent PDF that time. Remembering these cards were used already in gridpack.

Reply:
I don't really recall the time when 316200 was the recommended PDF set among other NNPDF31 releases... 306000 was for a few years when NNPDF31 first came out and then after some issues on negative weights were found 325300 became the nominal one.

If you do "$DEFAULT_PDF_SETS=lhaid" this will automatically fetch the recommended PDFs from [1].

[1] https://github.com/cms-sw/genproductions/tree/master/MetaData

Reply:
@sihyunjeon as informed by PdmV, 316200 was used in MC central production for the 2020 campaigns. 
The MetaData only shows 2016 and 2017 cases. Is it not updated to newer versions?

Reply:
Those 2016 and 2017 are just some flags for RunIISummer20 campaign's 2016 and 2017 data taking scenarios. And this is because (if I am correct again) CPX tunes were developed and used since 2017. Basically we have same scenarios in terms of hard process + showering/hadronization for 2017 and 2018 so that's why we don't have 2018 there.

For the RunIISummer20 campaign (==ultra legacy), it is recommended to follow the 2017 scenario (i.e. lhapdf=325300). And 316200 might be used for some cases which I am not aware of, but as far as I am aware, most of the samples follow 325300 recommendation.

For example, you can look at [1], EGM-RunIISummer20UL16wmLHEGEN-00003 at the top which was produced like 1-2 months ago, and as you can see it's **RunIISummer20** campaign for the DY sample. If you look at the "Fragment", you can see that it uses the gridpack in [2] which also uses 325300.

[1] https://cms-pdmv.cern.ch/mcm/requests?prepid=*RunIISummer20*wmLHEGEN*&dataset_name=DY*&page=0&shown=524415
[2] /cvmfs/cms.cern.ch/phys_generator/gridpacks/slc7_amd64_gcc700/13TeV/madgraph/V5_2.6.5/dy_fxfx_incl/dyellell012j_5f_NLO_FXFX_slc7_amd64_gcc700_CMSSW_10_6_19_tarball.tar.xz

Reply:
no problem. 325300 is Much better.